apb.yml

```diff
@@ -1,19 +1,22 @@
+---
 version: 1.0
 name: hello-world-apb
 description: deploys hello-world web application
-bindable: False
+bindable: "False"
```

Comment:
any reason these need to be strings and not bools?

Reply:
https://travis-ci.org/djzager/Hello-World-APB/jobs/339544060
```
apb.yml
  1:1       warning  missing document start "---"  (document-start)
  4:11      warning  truthy value is not quoted  (truthy)
  8:81      error    line too long (89 > 80 characters)  (line-length)
  14:11     warning  truthy value is not quoted  (truthy)
  17:81     error    line too long (88 > 80 characters)  (line-length)
```